Makefile

```diff
@@ -0,0 +1,51 @@
+GO ?= $(shell command -v go 2> /dev/null)
+MATTERMOST_CLOUD_IMAGE ?= mattermost/mattermost-cloud:test
+MACHINE = $(shell uname -m)
+BUILD_IMAGE = golang:1.12
+BASE_IMAGE = alpine:3.9
+GOPATH ?= $(shell go env GOPATH)
+GOFLAGS ?= $(GOFLAGS:)
+BUILD_TIME := $(shell date -u +%Y%m%d.%H%M%S)
+BUILD_HASH := $(shell git rev-parse HEAD)
+
+export GO111MODULE=on
+
+## Checks the code style, tests, builds and bundles.
+all: check-style dist
```

Comment:
We don't actually have a `test` target here -- intentional?

Reply:
will migrate that later

Reply:
Sounds good -- should we update the comment now, and add both later?